opennms-webapp/src/main/webapp/WEB-INF/jsp/element/service.jsp

```diff
@@ -314,7 +320,29 @@ function doDelete() {
       
             <!-- Recent outages box -->
             <jsp:include page="/outage/serviceOutages-box.htm" flush="false" />
-      </div> <!-- content-right -->
+
+          <div class="card">
+              <div class="card-header"><span>Application Perspective Monitoring</span></div>
+              <table class="table table-sm">
+                  <tr>
+                      <th>Perspective</th>
+                      <th>Polling Status</th>
+                      <th>Outage ID</th>
+                  </tr>
+                  <% ServiceJspUtil util = new ServiceJspUtil(service, outages);
+                      for(OnmsMonitoringLocation location : util.getAllPerspectives()) {
+                          Optional<OnmsOutage> outage = util.getOutageForPerspective(location);
+                  %>
+                  <tr>
+                      <td><i class="fa fa-server" aria-hidden="true"></i> <%=location.getLocationName()%></td>
```

Comment:
The icon has no benefit here.

Reply:
I added the icon since @indigo423 sugested to add it for consistency reasons. I see his point but have no strong preference either way. @indigo423 do you want to discuss with @fooker what we want to show here?

Reply:
As this caused a lot of confusion in the last meetings, this should be removed for now.

Reply:
ok, removed.